### PR TITLE
Feature: handle PoS status

### LINKED ISSUE #1027: Smeshing tab is not updateing after deleting POS data
### Environment

* Smapp version: 0.xx.x | develop
* Node version: v0.x.xx 
* OS: Windows 10 | macOS High Sierra (10.13) | Ubuntu 20.04

### Describe the bug

When POS data deleted , the smashing tab is not updating

### Steps to reproduce

https://user-images.githubusercontent.com/54288612/203847663-bf44bbda-18ec-4168-b19e-add8f35283d2.mov

### Expected behavior

The smeshing screen is ready for new POS data folder

### Actual behavior
The smeshing tab is freezing on previous state

### PR DESCRIPTION
**Pull request by maparr**
It closes #1027 

Screenshot 2022-12-15 at 00 10 15
Screenshot 2022-12-15 at 00 09 23

### CODE REVIEW

File: proto/spacemesh/v1/PostSetupStatus.ts [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
 
 import type { PostSetupOpts as _spacemesh_v1_PostSetupOpts, PostSetupOpts__Output as _spacemesh_v1_PostSetupOpts__Output } from '../../spacemesh/v1/PostSetupOpts';
-import type { Long } from '@grpc/proto-loader';
 
 // Original file: proto/smesher_types.proto
 
 export enum _spacemesh_v1_PostSetupStatus_State {
-  STATE_UNSPECIFIED = 0,
-  STATE_NOT_STARTED = 1,
+  STATE_NOT_STARTED = 1, // resume . (I stooped I can resume  |> there is no PoS data, u have to wait) or node is not init PoST
```
Comment (brusherru) [inline proto/spacemesh/v1/PostSetupStatus.ts]: Beware of upgrading .ts implementations manually.

The proper way is to update .proto files (if you don't like to copy files — we can replace it as a submodule, then you'll need only to bump the version in the file & re-generate proto files.

So, update the proto files to the ones used by go-spacemesh (v1.5.3). And run `yarn build:proto`

File: desktop/NodeManager.ts [hunk 2]
```diff
@@ -459,7 +459,8 @@ class NodeManager {
     if (!this.nodeProcess) return;
     try {
       // Request Node shutdown
-      await this.nodeService.shutdown();
+      this.nodeProcess.kill('SIGKILL');
```
Comment (brusherru) [inline desktop/NodeManager.ts]: Wait. It should not be so rough.
Send `SIGTERM` instead.

```suggestion
      this.nodeProcess.kill('SIGTERM');
```

**Review by brusherru — CHANGES_REQUESTED** [review]
The rest seems OK, but still can't test it entirely.
Having some problems with syncing the latest node 🤔

File: proto/spacemesh/v1/PostSetupStatus.ts (hunk 1 above)
Comment (maparr) [reply proto/spacemesh/v1/PostSetupStatus.ts]: @brusherru , yes, for sure, waiting for the Proto files update, go-spacemesh release, and network.

Comment (brusherru) [reply proto/spacemesh/v1/PostSetupStatus.ts]: Proto files are already there: https://github.com/spacemeshos/api/blob/51db8aeb7ef5bad8f00a9e304a75b004bf0a5978/spacemesh/v1/smesher_types.proto#L120-L132. You just need to copy-paste the entire file and build ts-files :)

**Review by brusherru — APPROVED** [review]
👍

**Review by brusherru — CHANGES_REQUESTED** [review]
Wait-wait :)

Probably it is a draft and that's why you made a couple of strange commits :)
I mean this https://github.com/spacemeshos/smapp/pull/1047/files#diff-a67f63211f160c9d04983b5160ea7a39d5e30acceeea8bbef04106eabdbfe172R58-R62 and what is the purpose of https://github.com/spacemeshos/smapp/pull/1047/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R23 ?

I believe there should be no [skip ci] commits in the branch that is going to be merged

**Comment by maparr** [comment]
@brusherru , the second one about Windows OS, is a way how to avoid the issue with go version check verification with custom build on the Windows, will be documented in our wiki pages

and first, will be removed, I'm checking rewards on network 997 and checking details. The comment will be removed, just testing is an issue will be with it or not, will update u here to check details .

**Comment by maparr** [comment]
Working with 997 netowrk

**Comment by maparr** [comment]
Hi @monikasmolarek , can u check this solution on Windows and MacOS, go-spacemesh commit 45e5b646f83c3f5a54d13091bd9306f146cd9ede and network 997

**Review by brusherru — APPROVED** [review]
👍

**Comment by monikasmolarek** [comment]
Works as expected on Windows and Mac. Creating, editing, deleting PoS all correct, rewards also correctly received, although shown in Transactions log only after restarting Smapp.

**Pull request merged**